apps/dokuha-roadmap-frontend/src/app/list/page.tsx

```diff
@@ -0,0 +1,113 @@
+'use client'
+
+import Button from "@/components/ui/Button";
+import { useEffect, useState } from "react";
+
+export type LearningContent = {
+    id: string;
+    userId: string;
+    title: string;
+    totalPage: number;
+    currentPage: number;
+    note: string;
+    createdAt: string;
+    updatedAt: string;
+};
+
+export default function list(){
+
+  const [contents, setContents] = useState<LearningContent[]>([]);
+
+  const userId = '00000000-0000-0000-0000-000000000001';
+  useEffect(()=>{
+    const fetchUsers = async () => {
+      try {
+        const response = await fetch(`${process.env.NEXT_PUBLIC_GET_CONTENTS_URL}?userId=${userId}`);
+
+        if (!response.ok) {
+          throw new Error(`HTTP error! status: ${response.status}`);
+        }
+
+        const data = await response.json();
+        console.log(data);
+        setContents(data.data);
+      } catch (error) {
+        console.log('エラーが発生しました:', error);
```

Comment:
エラーハンドリングも適当なメッセージ入れて後回し。って pending事項 に書いてくれていて助かった！
ので、TODO コメントだけ残しておいてほしい〜
```suggestion
        // TODO: 適切なエラーハンドリングに置き換える
        console.error('エラーが発生しました:', error);
```